doc/source/getting_started.rst

```diff
@@ -28,180 +29,72 @@ To test that everything works properly, open up IPython::
-
-If you just want to start using the latest development version (the most up to date)::
+For the newest features (and potentially more bugs), you can instead install
+the development version by:
 
     pip install -e git+https://github.com/mne-tools/mne-python#egg=mne-dev
 
-.. _check_and_optimize:
+If you plan to contribute to the project, please follow the git instructions: 
+:ref:`using-git`
 
-Check and Optimize Installation
--------------------------------
+Checking your installation
+--------------------------
 
 To check that everything went fine, in ipython, type::
 
     >>> import mne
 
-If you get a new prompt with no error messages, you should be good to go.
-
-CUDA Optimization
-^^^^^^^^^^^^^^^^^
-
-If you want to use NVIDIA CUDA for filtering (can yield 3-4x speedups), you'll
-need to install the NVIDIA toolkit on your system, and then both pycuda and
-scikits.cuda, see:
-
-https://developer.nvidia.com/cuda-downloads
-
-http://mathema.tician.de/software/pycuda
-
-http://wiki.tiker.net/PyCuda/Installation/
-
-https://github.com/lebedov/scikits.cuda
-
-To initialize mne-python cuda support, after installing these dependencies
-and running their associated unit tests (to ensure your installation is correct)
-you can run:
-
-    >>> mne.cuda.init_cuda() # doctest: +SKIP
-
-If you have everything installed correctly, you should see an INFO-level log
-message telling you your CUDA hardware's available memory. To have CUDA
-initialized on startup, you can do:
-
-    >>> mne.utils.set_config('MNE_USE_CUDA', 'true') # doctest: +SKIP
-
-You can test if MNE CUDA support is working by running the associated test:
-
-    nosetests mne/tests/test_filter.py
-
-If all tests pass with none skipped, then mne-python CUDA support works.
-
-
-.. _detailed_notes:
-
-Detailed Notes
---------------
-
-MNE is written in pure Python making it easy to setup on
-any machine with Python >=2.6, NumPy >= 1.6, SciPy >= 0.7.2
-and matplotlib >= 1.1.0.
-
-Some isolated functions (e.g. filtering with firwin2) require SciPy >= 0.9.
-
-To run all documentation examples the following additional packages are required:
-
-    * PySurfer (for visualization of source estimates on cortical surfaces)
-
-    * scikit-learn (for supervised and unsupervised machine learning functionality)
-
-    * pandas >= 0.8 (for export to tabular data structures like excel files)
-
-    * h5py (for reading and writing HDF5-formatted files)
+If you get a new prompt with no error messages, you should be good to go!
+Consider reading the :ref:`detailed_notes` for more advanced options and
+speed-related enhancements.
 
-Note. For optimal performance we recommend installing recent versions of
-NumPy (> 1.7), SciPy (> 0.10) and scikit-learn (>= 0.14).
+mne-python basics
+-----------------
+mne-python uses its own custom objects to store M/EEG data. A full description
+of these objects is available in the :ref:`api_reference` section.
+A typical MNE-python workflow is as follows:
```

Comment:
I lean toward `mne-python` since I'm most used to seeing that, but I think in the paper we did capitals somewhere. Do you have a preference?